src/main.rs

```diff
@@ -54,6 +54,21 @@ struct Opts {
         default = "false"
     )]
     hard_linebreaks: bool,
+
+    #[options(
+        no_short, 
+        long="front-matter-inclusion-key",
```

Comment:
I haven't come up with a better flag yet, but I'm not 100% happy with the naming here. My main concerns comes from the fact this does two things (it both sets the key to look for, and also toggles the behavior of only including notes with that key set) however the naming implies it only does the former.

Perhaps those two things should in fact be two separate arguments?